src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -929,11 +939,21 @@ private List<String> buildArrangements(Map<String, Object> pairs, boolean isSign
 
     }
 
-    private List<SscsDocument> buildDocumentsFromData(List<InputScannedDoc> records) {
+    private List<SscsDocument> buildDocumentsFromData(List<InputScannedDoc> records, boolean formTypeUpdated, String orgFormType, String newFormType) {
         List<SscsDocument> documentDetails = new ArrayList<>();
         if (records != null) {
             for (InputScannedDoc record : records) {
 
+                String fromType = record.getSubtype();
```

Comment:
variable name should be formType?

Reply:
thank you

Reply:
This still says fromType
![image](https://user-images.githubusercontent.com/110455092/190418568-4d1a4797-11a6-4580-b63b-b31ea47267fe.png)

Reply:
oops, my bad, another one push was fail, just updated